Content/Guides/syslog-ng-guide-admin/rewrite-set-matches.htm

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<html xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd">
+    <head>
+        <meta name="description" content="" />
+    </head>
+    <body name="rewrite-set-matches">
+        <h1 name="rewrite-set-matches">Setting match variables with the set-matches() rewrite rule</h1>
+        <MadCap:keyword term="rewrite()">
+        </MadCap:keyword>
+        <MadCap:keyword term="set-matches()">
+        </MadCap:keyword>
+        <p>Match macros (<span class="Code">$1, $2, ... $255</span>) are temporary variables. They can be used for general purposes when operating with list-like items. For example, the <MadCap:xref href="regular-expressions.htm"><span style="color: #04aada;" class="mcFormatColor">match() filter</span></MadCap:xref> stores capture group results in match variables when the <span class="Code">store-matches</span> flag is set, or the <MadCap:xref href="json-parser.htm"><span style="color: #04aada;" class="mcFormatColor">json-parser()</span></MadCap:xref> produces match variables if the parsed JSON data is an array.</p>
+        <p>It is possible to set match variables in a single operation with the <span class="Code">set-matches()</span> rewrite function. <span class="Code">set-matches()</span> uses <MadCap:variable name="General.abbrev"></MadCap:variable> list expressions to set <span class="Code">$1, $2, ... $255</span>, so it can be considered as a conversion function between <MadCap:variable name="General.abbrev"></MadCap:variable> lists and match variables.</p>
+        <div class="Note">
+            <p class="Hyphenation"><span class="AllNoteStyles">NOTE:</span> The <span class="Code">$*</span> macro can be used to convert match variables into a <MadCap:variable name="General.abbrev"></MadCap:variable> list, which can be further manipulated using <MadCap:xref href="reference-template-functions.htm#template-function-list"><span style="color: #04aada;" class="mcFormatColor">list template functions</span></MadCap:xref>, or turned into a list in type-aware destinations.</p>
+        </div>
+        <div class="Note">
+            <p class="Hyphenation">
+            <span class="AllNoteStyles">TIP:</span> The <span class="Code">unset-matches()</span> rewrite rule can be used to reset match variables to be empty.</p>
+        </div>
+
+        <p></p>
+        <div>
+            <h6>Declaration</h6><pre>rewrite &lt;name_of_the_rule&gt; {
+    set-matches("&lt;list-expression or list-based template function&gt;");
+};</pre>
+        </div>
+        <div class="Example">
+            <h6>Example usage for the set-matches() rewrite function</h6>
+            <p><b>Example using string:</b>
+            </p><pre>rewrite {
```

Comment:
Ide szeretnék kérni a "p" után és a "pre" elé még egy "p"-t amiben leírjuk a usernek hogy ha konkrétan ezt a példát berakja a confba akkor konkrétan ezekkel a valuekkal amiket itt használunk mi történik. mittomén, "in the following example, the <b class="Code">foo</b>, <b class="Code">bar</b> and <b class="Code">baz</b> values do this or that."